google/cloud/iot_v1/services/device_manager/transports/base.py

```diff
@@ -39,15 +38,6 @@
 except pkg_resources.DistributionNotFound:
     DEFAULT_CLIENT_INFO = gapic_v1.client_info.ClientInfo()
 
-try:
-    # google.auth.__version__ was added in 1.26.0
-    _GOOGLE_AUTH_VERSION = google.auth.__version__
-except AttributeError:
-    try:  # try pkg_resources if it is available
-        _GOOGLE_AUTH_VERSION = pkg_resources.get_distribution("google-auth").version
-    except pkg_resources.DistributionNotFound:  # pragma: NO COVER
-        _GOOGLE_AUTH_VERSION = None
```

Comment:
This change implies that the transitive dependency on `google-auth-library-python > 1.25.x` has arrived:  if that were so (it is not, AFAIK), then the pin in `testing/constraints-3.6.txt` would need to be adjusted as well.